docs/source/daml/intro/2_Scenario.rst

```diff
@@ -1,82 +1,84 @@
 
   More on ``Actions`` and ``do`` blocks in :doc:`5_Restrictions`.
 
 
   If that doesn't quite make sense yet, for the time being you can think of this arrow as extracting the right-hand-side value from the ledger and storing it into the variable on the left.
-- The argument ``"Alice"`` to ``getParty`` does not have to be enclosed in brackets. Functions in DAML are called using the syntax ``fn arg1 arg2 arg3``.
+- The argument ``"Alice"`` to ``allocateParty`` does not have to be enclosed in brackets. Functions in DAML are called using the syntax ``fn arg1 arg2 arg3``.
 
 With a variable ``alice`` of type ``Party`` in hand, you can submit your first transaction. Unsurprisingly, you do this using the ``submit`` function. ``submit`` takes two arguments: a ``Party`` and an ``Update``.
 
-Just like ``Scenario`` is a recipe for a test, ``Update`` is a recipe for a transaction. ``create Token with owner = alice`` is an ``Update``, which translates to the transaction creating a ``Token`` with owner Alice.
+Just like ``Script`` is a recipe for a test, ``Commands`` is a recipe for a transaction. ``createCmd Token with owner = alice`` is a ``Commands``, which translates to a list of commands that will be submitted to the ledger creating a transaction which creates a ``Token`` with owner Alice.
 
 You'll learn all about the syntax ``Token with owner = alice`` in :doc:`3_Data`.
 
-You could write this as ``submit alice (create Token with owner = alice)``, but just like scenarios, you can assemble updates using ``do`` blocks. A ``do`` block always takes the value of the last statement within it so the syntax shown in the scenario above gives the same result, whilst being easier to read.
+You could write this as ``submit alice (createCmd Token with owner = alice)``, but just like scripts, you can assemble commands using ``do`` blocks. A ``do`` block always takes the value of the last statement within it so the syntax shown in the commands above gives the same result, whilst being easier to read. Note however, that the commands submitted as part of a transaction are not allowed to depend n each other.
 
-Running scenarios
------------------
+Running scripts
+---------------
 
-There are two ways to run scenarios:
+There are a few ways to run DAML Scripts:
 
-- In DAML Studio, providing visualizations of the resulting ledger
-- Using the command line, useful for continuous integration
+- In DAML Studio against a test ledger, providing visualizations of the resulting ledger
+- Using the command line ``daml test`` also against a test ledger, useful for continuous integration
```

Comment:
Is this true? I thought `daml test` only runs scenarios, and one has to use `daml test-script` to test scripts?

Reply:
Yes it is, `daml test` works against scripts and runs them in the scenario runner. `daml test-script` runs against an actual ledger (unfortunate naming).